Makefile

```diff
@@ -18,9 +18,13 @@ CGO_ENABLED=0
 GOOS=linux
 
 build:
-	./hack/build.sh
+	./hack/build.sh -f
```

Comment:
Maybe we should add somewhere a `build.sh --codegen` as this will do the formatting/doc gen/license check ?

Not sure whether this is the proper place.

Reply:
I expect this to happen upstream only, given the resource constraints in CI, I am hoping we do only two operations in OpenShift CI which are relevant
1. build the binary
2. Run e2e tests against openshift
thats it.